CONTRIBUTING.md

```diff
@@ -35,10 +35,11 @@ This SDK is linted with `Prettier` and `ESlint`.
 
 ### How to Release
 
-1. Draft New Release on GitHub.com
-2. Set tag to v0.0.0 following [Semver 2.0](https://semver.org/)
-3. Set title to "v0.0.0 Major Feature Name"
-4. Set contents to follow [KeepAChangeLog.com 1.0](https://keepachangelog.com/en/1.0.0/), but limited to just the new release information
+1. Commit version update in package.json following [Semver 2.0](https://semver.org/)
+2. Draft New Release on GitHub.com
+3. Set tag to 0.0.0 (matching what is in the package.json)
+4. Set title to "v0.0.0 Major Feature Name"
```

Comment:
Sort of the same issue here. v0.0.0 doesn't look enough like a placeholder to me